resource/reactivedrop_schinese.txt

```diff
@@ -10379,14 +10379,14 @@
-		"rd_crafting_beta2_signup_explanation"		"If you join the Anomalous Materials Project - Phase 2, the following things will happen:\n• You will be authorized to find and collect anomalous materials during missions. If you are already authorized, your previous authorization remains in effect.\n• Your Steam account ID will be added to a list that we will use to compute some statistics about how people interact with the crafting materials system.\n• You will receive a box containing one Strange weapon of your choice. Strange weapons have the same stats, appearance, and requirements as their non-Strange counterparts, but are compatible with \"Strange Devices\", tiny computers that track and display statistics about your weapon.\n• You will receive two boxes each containing a Strange Device of your choice. These are in addition to any boxes you received during previous phases of the project.\n• You will receive a bonus set of crafting material locations (as if you had played for six hours).\n• You are encouraged (but not required) to give feedback about your experience with the Anomalous Materials Project on the Steam discussions forum for Alien Swarm: Reactive Drop or on Discord.\n\nThe following things will happen regardless of your choice:\n• Since January 2024, you have been passively finding a large quantity of crafting material locations by playing at least 30 minutes in a week and a smaller quantity of crafting material locations for every 6 total hours of play-time. You will continue to passively find these locations regardless of your participation in this beta."
+"[english]rd_crafting_beta2_signup_title"		"Anomalous Materials Project - Phase 2"
+"rd_crafting_beta2_signup_title"		"异常材料项目 - 第2阶段"
+// this is a letter from Helvetica Scenario to the player (commander).
+// the "office" she is referring to is the Office of Consensus Maintenance, her employer.
+// %s1 is the player's display name.
+// TODO: "rd_crafting_beta2_signup_flavor""指挥官 %s1 同志，\n\n\n\n祝好，\n赫尔维提卡•西纳瑞欧博士\n生态一致性部\n共识维护办公室 "
```

Comment:
```suggestion
// TODO: "rd_crafting_beta2_signup_flavor""Cmdr. %s1,\n\n\n\nBest,\nDr. Helvetica Scenario\nEcological Consistency Dept.\nOffice of Consensus Maintenance"
```

It's too early for this line to be translated, the continues integration process doesn't like comments to be translated.